src/anomalib/models/video/ai_vad/lightning_model.py

```diff
@@ -138,12 +141,14 @@ def validation_step(self, batch: dict[str, str | torch.Tensor], *args, **kwargs)
         """
         del args, kwargs  # Unused arguments.
 
-        boxes, anomaly_scores, image_scores = self.model(batch["image"])
-        batch["pred_boxes"] = [box.int() for box in boxes]
-        batch["box_scores"] = [score.to(self.device) for score in anomaly_scores]
-        batch["pred_scores"] = torch.Tensor(image_scores).to(self.device)
+        predictions = self.model(batch.image)
 
-        return batch
+        return replace(
```

Comment:
would it be possible to have a method in `Batch` like `update` or `replace` that updates/replaces these fields and return `self`? This way a user/implementer would not need to import `from dataclasses import replace` every time to update/replace the batch?